app/views/foreman_ansible/job_templates/convert_to_rhel.erb

```diff
@@ -44,5 +63,6 @@ kind: job_template
       reboot:
         reboot_timeout: 1800
 <%- end -%>
+# Releated to Satellite
```

Comment:
the comment would be more descriptive as "This will update system facts in Satellite and link the correct OS of the host" or something similar. Also if this comment is mainly for developers, it would be better to keep it in erb, otherwise it ends up in the resulting playbook

<%- # comment -%>